tests/formattedcode/data/cyclonedx/simple-icu-expected.json

```diff
@@ -0,0 +1,36 @@
LicenseRef-scancode-unicode)', 'license_detections': [{'license_expression': 'unicode', 'detection_log': ['from-package-file'], 'matches': [{'score': 100.0, 'start_line': 7, 'end_line': 7, 'matched_length': 10, 'match_coverage': 100.0, 'matcher': '2-aho', 'license_expression': 'unicode', 'rule_identifier': 'unicode_12.RULE', 'rule_url': 'https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/unicode_12.RULE', 'matched_text': 'License & terms of use: http://www.unicode.org/copyright.html'}]}, {'license_expression': 'fsf-free', 'detection_log': ['from-package-file'], 'matches': [{'score': 100.0, 'start_line': 13, 'end_line': 14, 'matched_length': 18, 'match_coverage': 100.0, 'matcher': '2-aho', 'license_expression': 'fsf-free', 'rule_identifier': 'fsf-free.LICENSE', 'rule_url': 'https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/fsf-free.LICENSE', 'matched_text': 'This configure script is free software; the Free Software Foundation\\n# gives unlimited permission to copy, distribute and modify it.'}]}, {'license_expression': 'fsf-free AND unicode', 'detection_log': ['from-package-file'], 'matches': [{'score': 100.0, 'start_line': 1587, 'end_line': 1588, 'matched_length': 18, 'match_coverage': 100.0, 'matcher': '2-aho', 'license_expression': 'fsf-free', 'rule_identifier': 'fsf-free.LICENSE', 'rule_url': 'https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/fsf-free.LICENSE', 'matched_text': 'This configure script is free software; the Free Software Foundation\\ngives unlimited permission to copy, distribute and modify it.'}, {'score': 100.0, 'start_line': 1590, 'end_line': 1590, 'matched_length': 10, 'match_coverage': 100.0, 'matcher': '2-aho', 'license_expression': 'unicode', 'rule_identifier': 'unicode_12.RULE', 'rule_url': 'https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/unicode_12.RULE', 'matched_text': 'License & terms of use: http://www.unicode.org/copyright.html'}]}, {'license_expression': 'fsf-free', 'detection_log': ['from-package-file'], 'matches': [{'score': 100.0, 'start_line': 8663, 'end_line': 8664, 'matched_length': 19, 'match_coverage': 100.0, 'matcher': '2-aho', 'license_expression': 'fsf-free', 'rule_identifier': 'fsf-free.RULE', 'rule_url': 'https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/fsf-free.RULE', 'matched_text': 'This config.status script is free software; the Free Software Foundation\\ngives unlimited permission to copy, distribute and modify it.\"'}]}], 'other_license_expression': None, 'other_license_expression_spdx': None, 'other_license_detections': [], 'extracted_license_statement': None, 'notice_text': None, 'source_packages': [], 'extra_data': {}, 'repository_homepage_url': None, 'repository_download_url': None, 'api_data_url': None, 'datafile_paths': ['simple-icu/source/configure'], 'datasource_ids': ['autotools_configure'], 'purl': 'pkg:autotools/source'}"
```

Comment:
Would there be a way to get something less verbose?

Reply:
Or should this work of top level packages and dependencies rather than from file-level package_data?

Reply:
yeah, maybe we can just use the `purl` and the `datafile_paths`?

Btw, this is working on the top-level packages only already.

Reply:
this is now just the following:

```
 "name": "WARNING",
 "value": "WARNING: component skipped in CycloneDX output: purl: pkg:autotools/source at datafile_paths: ['simple-icu/source/configure']"
```